core_weekly/template.py

```diff
@@ -283,6 +283,17 @@ def custom_sort(self, data, key_order):
 
         return sorted_dict
 
+    def verify_no_unknown_branch(self, grouped_data_by_branches, branches):
+        """Verify there is no unknown branch
+        :param grouped_data_by_branches: Data we want to verify
+        :type grouped_data_by_branches: dict
+        :param branches: Core known branches
+        :type branches: str
+        """
+        for key in grouped_data_by_branches:
+            if key not in branches:
+                raise Exception('Cannot find branch ' + key + ' in configured branches')
```

Comment:
```suggestion
                raise Exception('Cannot find branch "{}" in configured branches'.format(key))
```
It Will be easier to read and understand if the branch contains spaces/special characters

Reply:
Nice one, thanks !